gnssanalysis/gn_utils.py

```diff
@@ -52,6 +52,54 @@ def get_filetype(path):
     return suffix
 
 
+def configure_logging(verbose: bool, output_logger: bool = False) -> _Union[_logging.Logger, None]:
+    """Configure the logger object with the level of verbosity requested and output if desired
+
+    :param bool verbose: Verbosity of logger object to use for encoding logging strings, True: DEBUG, False: INFO
+    :param bool output_logger: Flag to indicate whether to output the Logger object, defaults to False
+    :return _Union[_logging.Logger, None]: Return the logger object or None (based on output_logger)
```

Comment:
These could be switched to Python3.10+ type unions, i.e. `_logging.Logger | None`
```suggestion
    :return _logging.Logger | None: Return the logger object or None (based on output_logger)
```

Reply:
Replaced this and other cases where Union was used